app/database/accessor/breeze_buddy/blacklisted_numbers.py

```
@@ -0,0 +1,206 @@
+    """
+    if not phone_number:
+        return False
+
+    cache_key = _cache_key(phone_number, merchant_id)
+
+    try:
+        redis = await get_redis_service()
+        cached = await redis.get(cache_key)
+        if cached is not None:
+            return cached == "1"
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)